Documenting 413 telco core fixed issues (#58713)
openshift/openshift-docs: release_notes/ocp-4-13-release-notes.adoc

* When you include more than three failure domains in the `ControlPlaneMachineSet` definition, the load balancing algorithm does not prioritize existing control plane machines. If you add a fourth failure domain that is alphabetically higher in precedence than the existing three failure domains to the definition, the fourth failure domain takes precedence over any existing failure domains. This behavior can apply rolling forward updates to a control plane machine. You can prevent this issue by setting existing in-use failure domains to a higher precedence than the new and unused failure domains. This action stabilizes each control plane machine during the course of adding more than three failure domains to the definition. (link:https://issues.redhat.com/browse/OCPBUGS-11968[*OCPBUGS-11968*])

> * On a single-node OpenShift instance, rebooting without draining the node to remove all the running pods can cause issues with workload container recovery. After the reboot, the workload restarts before all the device plugins are ready, resulting in resources not being available or the workload running on the wrong NUMA node. The workaround is to restart the workload pods when all the device plugins have re-registered themselves during the reboot recovery procedure. (link:https://issues.redhat.com/browse/OCPBUGS-2180[*OCPBUGS-2180*])

Reviewer: ```suggestion
* On a {sno} instance, rebooting without draining the node to remove all of the running pods can cause issues with workload container recovery. After the reboot, the workload restarts before all the device plugins are ready, resulting in resources not being available or the workload running on the wrong NUMA node. The workaround is to restart the workload pods when all of the device plugins have re-registered themselves during the reboot recovery procedure. (link:https://issues.redhat.com/browse/OCPBUGS-2180[*OCPBUGS-2180*])
```

> * An error may occur when deleting a pod that uses a SR-IOV netdevice. This error is caused by a change in RHEL 9 where the previous name of a network interface is added to its alternative names list when it is renamed. As a consequence, when a pod attached to an SR-IOV virtual function (VF) is deleted, the VF returns to the pool with a new unexpected name for example `dev69` instead of its original name for example `ensf0v2`. Although this error is non-fatal, Multus and SR-IOV logs may show the error while the system recovers on its own. Deleting the pod may take a few extra seconds due to this error. (link:https://issues.redhat.com/browse/OCPBUGS-11281[*OCPBUGS-11281*])

Reviewer: ```suggestion
* An error might occur when deleting a pod that uses an SR-IOV netdevice. This error is caused by a change in {op-system-base} 9 where the previous name of a network interface is added to its alternative names list when it is renamed. As a consequence, when a pod attached to an SR-IOV virtual function (VF) is deleted, the VF returns to the pool with a new unexpected name, for example, `dev69`, instead of its original name, for example, `ensf0v2`. Although this error is non-fatal, Multus and SR-IOV logs might show the error while the system recovers on its own. Deleting the pod might take a few extra seconds due to this error. (link:https://issues.redhat.com/browse/OCPBUGS-11281[*OCPBUGS-11281*])
```

* An incorrect priority class name and syntax error in the YAML definition of the daemon set responsible for updating the interface-specific safe sysctl is preventing the modification of the safe sysctl list for interfaces by using the cni-sysctl-allowlist Config Map in the `openshift-multus` namespace.
+
Workaround: Manually or by using a daemon set, modify the file `/etc/cni/tuning/allowlist.conf` on the nodes to address this issue. (link:https://issues.redhat.com/browse/OCPBUGS-11046[*OCPBUGS-11046*])

> * A new feature introduced in OCP 4.12 that enables UDP GRO also causes all veth devices to have one RX queue per available CPU (previously each veth had one queue). Those queues are dynamically configured by OVN and there is no synchronization between latency tuning and this queue creation. The latency tuning logic monitors the veth NIC creation events and starts configuring the RPS queue cpu masks before all the queues are properly created. This means that some of the RPS queue masks are not configured. Since not all NIC queues are configured properly there is a chance of latency spikes in a real-time application that uses timing-sensitive cpus for communicating with services in other containers. Applications that do not use kernel networking stack are not affected. (link:https://issues.redhat.com/browse/OCPBUGS-4194[*OCPBUGS-4194*])

Reviewer: I don't think we're allowed to use the OCP abbreviation in official docs. s/ {product-title} 

Might want to define the first use of OVN in the second sentence. 

CPU should be capitalized (or, at least it's most commonly capitalized in our docs). x2 instances of this.

> * A new feature introduced in OCP 4.12 that enables UDP GRO also causes all veth devices to have one RX queue per available CPU (previously each veth had one queue). Those queues are dynamically configured by OVN and there is no synchronization between latency tuning and this queue creation. The latency tuning logic monitors the veth NIC creation events and starts configuring the RPS queue cpu masks before all the queues are properly created. This means that some of the RPS queue masks are not configured. Since not all NIC queues are configured properly there is a chance of latency spikes in a real-time application that uses timing-sensitive cpus for communicating with services in other containers. Applications that do not use kernel networking stack are not affected. (link:https://issues.redhat.com/browse/OCPBUGS-4194[*OCPBUGS-4194*])

Reviewer: ```suggestion
* A new feature introduced in {product-title} 4.12 that enables UDP GRO also causes all veth devices to have one RX queue per available CPU (previously each veth had one queue). Those queues are dynamically configured by Open Virtual Network, and there is no synchronization between latency tuning and this queue creation. The latency tuning logic monitors the veth NIC creation events and starts configuring the RPS queue CPU masks before all the queues are properly created. This means that some of the RPS queue masks are not configured. Since not all NIC queues are configured properly there is a chance of latency spikes in a real-time application that uses timing-sensitive CPUs for communicating with services in other containers. Applications that do not use kernel networking stack are not affected. (link:https://issues.redhat.com/browse/OCPBUGS-4194[*OCPBUGS-4194*])
```
